AnnoyancesFilter/Cookies/sections/cookies_general.txt

```diff
@@ -1091,6 +1091,7 @@
 ###wt-cookie-box
 ###ww-cookie-information
 ###zaz-nb-plugin-cookie-message
+###iai_cookie
```

Comment:
Adding just this rule does not handle scroll locking.
Please use a CSS rule or scriptlet.